catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/FormulaTest.java

```diff
@@ -49,19 +53,23 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
 
-@RunWith(JUnit4.class)
+@RunWith(MockitoJUnitRunner.class)
 public class FormulaTest {
 
 	private List<InternToken> internTokenList;
 
 	@Before
 	public void setUp() {
-		internTokenList = new LinkedList<InternToken>();
+		internTokenList = new LinkedList<>();
+		if (ProjectManager.getInstance() == null) {
```

Comment:
Please remove unused Singleton reference to Projectmanager.

Reply:
Unfortunately, this 💩 is needed if you want to execute the test locally... Ask @ThomasSchwengler, he discovered this. 
Maybe it has something to do with Dagger 🤔